src/devices/onesti.ts

```diff
@@ -9,6 +9,24 @@ const e = exposes.presets;
 const ea = exposes.access;
 
 const fzLocal = {
+    nimly_lock_commandPinCodeRsp: {
+        cluster: 'closuresDoorLock',
+        type: ['commandSetPinCodeRsp','commandClearPinCodeRsp'],
+        convert: (model, msg, publish, options, meta) => {
+            const attributes: KeyValue = {};
+            if (msg.data.status === 0) {
+                if (msg.type === 'commandSetPinCodeRsp') {
+                    attributes.Last_successful_Pincode_Save = Date.now();
```

Comment:
```suggestion
                    attributes.last_successful_pincode_save = Date.now();
```

Also for below and make sure to update the expose